Bottom: 9c34f04abbb60db8a33ca6fdcc7b6c4b4b6c92b3
Top:    d651ce30e97658f91a43837de56ecfdc5d5ceb32
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-02 10:50:07 +0800

Refresh of vhost-prefetch-desc-indices

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 74148fd..34d38c2 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1909,7 +1909,7 @@ static int get_indirect(struct vhost_virtqueue *vq,
 }
 
 int vhost_prefetch_desc_indices(struct vhost_virtqueue *vq, __virtio16 *indices,
-				int num)
+			        int num)
 {
 	u16 last_avail_idx, total;
 	__virtio16 avail_idx;
